static/js/zamboni/validator.js

```diff
@@ -384,9 +384,12 @@ function initValidator($doc) {
 
     CompatMsgVisitor.prototype.message = function(msg) {
         if (msg.for_appversions) {
-            if (this.hasMatchingAppVer(msg.for_appversions)) {
-                var app = {guid: guid, version: version, id: id};
-                msg.tier = id;  // change the tier to match app/version
+            var guid = this.findMatchingApp(msg.for_appversions)
```

Comment:
I know commas aren't needed, but we're using them throughout the code, so I think I'd rather have one at the end of this line.